main.tf

```diff
@@ -93,11 +93,13 @@ module "olm_subscriptions" {
   count  = var.enable_olm ? 1 : 0
   source = "./modules/olm-subscriptions"
 
-  olm_namespace     = var.olm_namespace
-  install_namespace = var.olm_install_namespace
-  settings          = var.olm_subscription_settings
-  registry          = var.olm_registry
-  values            = var.olm_subscription_values
+  olm_namespace        = var.olm_namespace
+  install_namespace    = var.olm_install_namespace
+  enable_istio         = var.olm_use_istio
```

Comment:
How about using the same name here (both (`enable_istio` and `olm_enable_istio`) or (`use_istio` or `olm_use_istio`))?

Reply:
I had some ideas there, but yeah I will do that.